pandas/io/pickle.py

```diff
@@ -33,7 +34,27 @@ def to_pickle(obj, path, compression='infer', protocol=pkl.HIGHEST_PROTOCOL):
         .. [1] https://docs.python.org/3/library/pickle.html
         .. versionadded:: 0.21.0
 
-
+    Examples
+    --------
+    >>> from pandas import DataFrame, read_pickle
```

Comment:
you don't need these imports, instead use ``pd.DataFrame`` and ``pd.read_pickle``

Reply:
fixed 👍